.github/workflows/aws-deploy.yaml

```diff
@@ -27,7 +27,17 @@ jobs:
           aws-region: us-east-1
           role-to-assume: arn:aws:iam::728882028485:role/workflows-nextflow-ci-service-account-ServiceRole-CTLXKJIOCRO4
           role-duration-seconds: 1200
-      - name: Deploy 'prod' templates
+      - name: Deploy common configuration
+        uses: Sage-Bionetworks/sceptre-action@v1.1
+        with:
+          sceptre_version: 2.4.0
+          sceptre_subcommand: launch --yes common
+      - name: Deploy develop configuration
+        uses: Sage-Bionetworks/sceptre-action@v1.1
+        with:
+          sceptre_version: 2.4.0
+          sceptre_subcommand: launch --yes develop
```

Comment:
i believe this will fail if you don't have a `config/develop` folder and files to deploy.

Reply:
@tthyer: You can leave this block commented out. I have a PR in mind that I can open once this is merged that would only deploy to the dev account.

Reply:
@zaro0508: I just realized that Tess renamed one of the `prod` files to `config/develop/s3-cfn-templates.yaml`, so we should be good.

Reply:
However, the `prod` folder is [empty](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/tree/tthyer/cicd-refactor/config) now, so that might fail.

Reply:
yup, and you don't need that empty `prod/config.yaml` file either.